test/docker-e2e/e2e_state_sync_upgrade_test.go

```diff
@@ -37,8 +37,8 @@ func (s *CelestiaTestSuite) TestStateSyncWithAppUpgrade() {
 	t.Logf("Phase 1: Starting the chain with %d validators", len(cfg.Genesis.Validators()))
 
 	const baseAppVersion = appconsts.Version - 1
-	const targetAppVersion = appconsts.Version
-	t.Logf("Starting chain with app version %d and will upgrade to %d.", baseAppVersion, targetAppVersion)
+	const binaryVersion = appconsts.Version
```

Comment:
[blocking] this isn't the binary version. 

There are two different versions:
- App version is `v5`, `v6`, etc. 
- Binary version is `v5.0.5`, `v5.0.2-arabica`, etc.

They track different things.

Reply:
a fix applied